spec/lib/http_spec.rb

```diff
@@ -430,6 +430,31 @@ def setsockopt(*args)
         expect(response.to_s).to eq("#{body}-deflated")
       end
     end
+
+    context "with :normalize_uri" do
+      module Normalizer
+        def self.call(uri)
+          uri
+        end
+      end
+
+      it "Use the defaul Uri normalizer when user does not use uri normalizer" do
```

Comment:
URI should be all caps. And topic of example should start with lowercase. Also typo in *defaul*.
In this particular example should be moved into default context, either here:

https://github.com/httprb/http/blob/1eae1557f1fb68408190345cb731fc32007754bd/spec/lib/http_spec.rb#L21

or here:

https://github.com/httprb/http/blob/1eae1557f1fb68408190345cb731fc32007754bd/spec/lib/http_spec.rb#L466

And should look like:

``` ruby
it "normalizes URI" do
  response = HTTP.get "#{dummy.endpoint}/hello world"
  expect(response.to_s).to eq("hello world")
end
```

others examples are OK to be left in here, just make them less "wordy"...